src/chip/src/components/Chip.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import React from 'react';
+
+import { Chip as MuiChip, makeStyles } from '@material-ui/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/pro-light-svg-icons';
+import Colors from './colors';
+import type { ChipProps } from './types';
+
+const DeleteIcon = <FontAwesomeIcon fixedWidth icon={faTimes} />;
+
+const actionStates = (backgroundColor) => ({
+  '&:hover, &:focus': {
+    backgroundColor
+  },
+  '&:active': {
+    backgroundColor
+  },
+});
+
+const useStyles = (color) => {
```

Comment:
The hover color for the delete icon is gray for all colors except `primary`, which doesn't look good, and it's a dark purple for `primary`, which also does not look good. Please update.

<img width="96" alt="Screen Shot 2021-03-22 at 1 16 48 PM" src="https://user-images.githubusercontent.com/440299/112052887-e5701e80-8b10-11eb-90ef-5c104b66eaaf.png">
<img width="76" alt="Screen Shot 2021-03-22 at 1 14 04 PM" src="https://user-images.githubusercontent.com/440299/112052899-e99c3c00-8b10-11eb-80c7-8d8de660cecc.png">